autopts/pybtp/types.py

```diff
@@ -502,3 +504,12 @@ class BIGEncryption:
     BROADCAST_CODE_REQUIRED = 0x01
     DECRYPTING = 0x02
     BAD_CODE = 0x03
+
+
+class TMAPRole(IntEnum):
+    CALL_GATEWAY = 0x01
+    CALL_TERMINAL = 0x02
+    UNICAST_MEDIA_SENDER = 0x04
+    UNICAST_MEDIA_RECEIVER = 0x08
+    BROADCAST_MEDIA_SENDER = 0x10
+    BROADCAST_MEDIA_RECEIVER = 0x20
```

Comment:
Given that this is a bitfield, and that we define and use `BIT` function in `defs.py`, we should consider doing the same here, to make it clearer that it is a bitfield. 

Additionally, defining this as an `IntEnum` would actually prevent it from being a bitfield, so I think that is the wrong approach here

Reply:
I re-did this part.